dashboard/modules/resource/tests/test_resource.py

```diff
@@ -0,0 +1,244 @@
+import time
+import ray
+import requests
+import pytest
+
+from ray.dashboard.tests.conftest import *  # noqa
+from ray._private.test_utils import (
+    format_web_url,
+    wait_until_server_available,
+)
+
+
+@ray.remote(num_cpus=1, memory=1000, resources={"custom_a": 1})
+def f():
+    while True:
+        time.sleep(10)
+
+
+@ray.remote(num_cpus=1, num_gpus=1)
+def g():
+    while True:
+        time.sleep(10)
+
+
+@ray.remote(num_cpus=1, resources={"custom_b": 1})
+class Actor:
+    pass
+
+
+@pytest.mark.parametrize(
+    "ray_start_cluster", [{"include_dashboard": True}], indirect=True
+)
+def test_resource_summary_and_usage(ray_start_cluster):
+    cluster = ray_start_cluster
+    cluster.add_node(num_cpus=32, num_gpus=4, resources={"custom_a": 16})
```

Comment:
Can you use smaller num_cpus? The CI only has 2 CPUs, and 32 CPUs will create 32 processes (in this case 64) which is unnecessary

Reply:
Good point!